docs/releasing/publishing.md

```diff
@@ -10,18 +10,33 @@ Developers should pair on releases. When remote working, it can be useful to be
 
 2. Run `nvm use` to make sure you're using the right version of Node.js and npm.
 
-3. Run `npm install` to make sure you have the latest dependencies installed.
+3. Run `npm ci` to make sure you have the exact dependencies installed.
 
 4. Create and check out a new branch (`release-[version-number]`). See the [versioning documentation](/docs/contributing/versioning.md) for more information.
 
 5. Update the [`CHANGELOG.md`](../../CHANGELOG.md) by:
 
-   - changing the 'Unreleased' heading to the new version-number and release-type - for example, '3.11.0 (Feature release)'
-   - adding a new 'Unreleased' heading above the new version-number and release-type, so users will know where to add PRs to the changelog
+   - changing the 'Unreleased' heading to the new version number and release type. For example, '3.11.0 (Feature release)'
+   - adding a new 'Unreleased' heading above the new version number and release type, so users will know where to add PRs to the changelog
+   - saving your changes
 
-6. Update [`package/package.json`](../../package/package.json) version with the new version-number.
+6. Apply the new version number by running:
 
-7. Save the changes. Do not commit.
+   ```
+   npm version <NEW VERSION NUMBER> --no-git-tag-version --workspace package
+   ```
+
+   This step will update the `package.json` and project `package-lock.json` files.
+
+   Do not commit the changes.
+
+7. Update browser data from ["Can I use"](https://caniuse.com) by running:
+
+   ```
+   npx update-browserslist-db@latest
```

Comment:
I've included this change from

* https://github.com/alphagov/govuk-frontend/pull/3555

Our support branches don't use Dependabot so PostCSS (Autoprefixer) [**.browserslistrc** queries](https://github.com/alphagov/govuk-frontend/blob/support/4.x/.browserslistrc) log warnings for out-of-date browser support data

```
> 0.1%
last 2 Chrome versions
last 2 Firefox versions
last 2 Edge versions
last 2 Samsung versions
```

Reply:
Although do we need to partially cherry pick our wider browser support from `main`?

* https://github.com/alphagov/govuk-frontend/pull/3728

```
> 0.1% in GB
last 6 Chrome versions
last 6 Firefox versions
last 6 Edge versions
last 2 Samsung versions
```

Reply:
According to the notes from dev catch-up, we said we'd keep the current list to keep the support branch in as similar a state as when things started moving away, so no need to update it.

Reply:
Thanks for confirming

Do we need to rethink the "Can I use" data too?

For both v4 and v5 any updates will affect Browserslist queries